test/api/matrix.test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
```

Comment:
think we can slot this test in `routing.test.js` as we have it nested as a Routing API in our docs

Reply:
@jsani-radar would the best way to do this be a separate `describe('Matrix', ())` block in the routing.test.js test suite? or should it be part of the Routing block? the origin and destinations in the Matrix API are formatted a bit differently so I'd have to create separate variables if the latter

Reply:
@jsani-radar went ahead and moved it into routing.test.js as described above, and all tests are passing